equinix/fabric_mapping_helper.go

```diff
@@ -438,6 +437,25 @@ func operationToTerra(operation *v4.ConnectionOperation) *schema.Set {
 	return operationSet
 }
 
+func networkOperationToTerra(operation *v4.NetworkOperation) *schema.Set {
```

Comment:
There is a large refactor in progress to enable us to gradually migrate the provider from the deprecated terraform-plugin-sdk to its replacement, terraform-plugin-framework.  One thing we've found during that refactor is that shared, private functions like this make it more difficult to refactor resources & data sources from the SDK to the framework.

Instead of adding new helpers to this file, I recommend creating a new package & code file under the `internal` directory.  For example, you could define this function as `NetworkOperationToTerra` in the file `internal/mapping/fabric.go`, specifying `package mapping` at the top of that file; then in files that use this helper you would need to import `github.com/equinix/terraform-provider-equinix/internal/mapping`, and you would call the function as `mapping.NetworkOperationToTerra(...)`.